src/components/Slider.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { SFC } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { default as SliderRN } from '@react-native-community/slider'; // tslint:disable-line
```

Comment:
I think we can remove this rule from TSlint completely

Reply:
I've removed it in this PR https://github.com/autyzm-pg/friendly-plans-react/pull/278